notebooks/1_training.ipynb

```diff
@@ -11,18 +11,27 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Setup"
+    "## Setup\n",
+    "\n",
+    "### Prepare the training data\n",
+    "\n",
+    "You'll need some training data. To use the residential load data from OpenEI, run the following in the base directory of this repository:\n",
+    "\n",
+    "```bash\n",
+    "wget https://data.openei.org/files/153/RESIDENTIAL_LOAD_DATA_E_PLUS_OUTPUT.zip\n",
```

Comment:
I've added this to make it easy to get the training data and put it in the correct place via copy'n'paste.